aviary/subsystems/propulsion/utils.py

```diff
@@ -396,16 +398,16 @@ def setup(self):
                 has_diag_partials=True,
             ),
             promotes_inputs=[
-                ('T0', Dynamic.Mission.TEMPERATURE),
-                ('mach', Dynamic.Mission.MACH),
+                ('T0', Dynamic.Atmosphere.TEMPERATURE),
+                ('mach', Dynamic.Atmosphere.MACH),
             ],
             promotes_outputs=['theta_T'],
         )
 
         self.add_subsystem(
             'uncorrection',
             om.ExecComp(
-                'uncorrected_data = corrected_data * (delta_T + theta_T**.5)',
+                'uncorrected_data = corrected_data * (delta_T * theta_T**.5)',
```

Comment:
Are you sure this + should be a *?

Reply:
The values coming out of our turboshaft model were behaving strangely, so Carl and I investigated and found that the original equation here was incorrect and `theta_T` is a multiplicative factor. Likely a typo when originally writing the equation